adhocracy-plus/assets/scss/components/_blocks.scss

```diff
@@ -193,12 +169,12 @@ $block-padding-md: 7.5 * $padding;
             padding: $block-padding-md;
         }
     }
+}
 
-    .block-col_cta-content {
-        @media screen and (min-width: $breakpoint-md) {
-            width: 70%;
-            margin: auto;
-        }
+.block-col_cta-content {
+    @media screen and (min-width: $breakpoint-md) {
+        width: 70%;
+        margin: auto;
```

Comment:
ensures ctas that have content are not wider then the content, does not effect standalone ctas